README.md

```diff
@@ -3,66 +3,79 @@ kindle-highlights
 
 A Ruby gem for collecting your Kindle highlights.
 
-### Install
+### Installation
 ```
 gem install kindle-highlights
 ```
 
-### Use
-```ruby
-require 'kindle_highlights'
+### Usage
 
-# pass in your Amazon credentials. Loads your books (not highlights)
-# on init, so might take a while
-kindle = KindleHighlights::Client.new("email.address@gmail.com", "password")
+1. In an interactive Ruby shell ([`irb`](http://ruby-doc.org/stdlib-2.0.0/libdoc/irb/rdoc/IRB.html)), run the first of several commands:
```

Comment:
Since this is a library, it’s not intended to be run with irb. (It *can* be, but mostly for playing around.) I’d rephrase this as:

```
1. Load the library:

    ```
    require 'kindle_highlights'
    ```
````